.code-samples.meilisearch.yaml

```
@@ -8,7 +8,7 @@ get_one_index_1: |-
 list_all_indexes_1: |-
   client.get_indexes()
 create_an_index_1: |-
-  client.create_index('movies', {'primaryKey': 'movie_id'})
+  client.create_index('movies', {'primaryKey': 'id'})
 update_an_index_1: |-
   client.index('movies').update(primary_key='movie_review_id')
```

```suggestion
  client.index('movies').update(primary_key='id')
```

The two syntaxes are very different, is it expected?